lib/improver/utilities/load.py

```diff
@@ -63,8 +64,14 @@ def load_cube(filepath, constraints=None, no_lazy_load=False):
     # Remove metadata prefix cube if present
     constraints = iris.Constraint(
         cube_func=lambda cube: cube.long_name != 'prefixes') & constraints
-    cube = iris.load_cube(filepath, constraint=constraints)
-
+    cubes = iris.load(filepath, constraints=constraints)
+    if len(cubes) == 0:
```

Comment:
Preferable to ```len(cubes) == 0``` is ```if not cubes```.
Alternatively you could reorder to have ```len(cubes) >1, len(cubes) == 1, else```

Reply:
Fixed